migrating_gazebo_classic_ros2_packages.md

```diff
@@ -0,0 +1,712 @@
+# Migrating ROS 2 packages that use Gazebo Classic
+
+The Gazebo simulator has its roots in the Gazebo Classic project, but it has a
+few significant differences that affect how a ROS 2 project uses the simulator.
+One difference is that ROS 2 projects now use the
+[ros_gz](https://github.com/gazebosim/ros_gz) package instead of
+[gazebo_ros_pkgs](https://github.com/ros-simulation/gazebo_ros_pkgs) as the
+source of launch files and other useful utilities. Another major difference is
+that while [gazebo_ros_pkgs](https://github.com/ros-simulation/gazebo_ros_pkgs)
+provided a set of plugins that directly get loaded by Gazebo Classic and run as
+part of the simulation to provide an interface between ROS and Gazebo Classic,
+[ros_gz](https://github.com/gazebosim/ros_gz) is primarily used as a bridge
+between ROS and gz-transport topics. Knowing these conceptual differences is
+important in making the transition.
+
+**Note:** Since the name of the project has gone through two major changes, we
+highly recommend you read the [history](https://gazebosim.org/about) of the
+project to have a better understanding of the terminology used in this tutorial
+and elsewhere. As a convention we refer to older versions of Gazebo, those with
+release numbers like Gazebo 9 and Gazebo 11 as "Gazebo Classic." Newer versions
+of Gazebo, formerly called "Ignition", with lettered releases names like
+Harmonic, are referred to as just "Gazebo" in this document. This tutorial will
+show how to migrate an existing ROS 2 package that uses the `gazebo_ros_pkgs`
+package to the new `ros_gz`. We will use the
+[turtlebot3_simulations](https://github.com/ROBOTIS-GIT/turtlebot3_simulations/)
+package as an example. The complete, migrated version of
+`turtlebot3_simulations` covered in this tutorial, can be found in
+[this fork](https://github.com/azeey/turtlebot3_simulations/tree/new_gazebo).
+
+We'll start by following the
+[PC Setup](https://emanual.robotis.com/docs/en/platform/turtlebot3/quick-start/)
+guide to install the necessary prerequisites for simulating Turtlebot3. This
+will install additional packages, such as Nav2 and Cartographer, which we will
```

Comment:
links to nav2 and Cartograoher Github ?

Reply:
Fixed in https://github.com/gazebosim/docs/pull/425/commits/9ec0798398db9d15922b57bcb41407cbce45159b. I realized the tutorial doesn't actually use these as this sentence promises, so I added a small section in https://github.com/gazebosim/docs/pull/425/commits/5b655cf6103676e428f3e3d0a3bbb7f65e36f693